### DISCUSSION TITLE
Access to my private repository

### DISCUSSION THREAD

**Discussion by casmatos** [community/community#40081] [Repositories] [answered]
### Select Topic Area

General

### Body

I try to open my private repository via web, but the page take too long to open. In my other repositories i can access them.

**Comment by WernerMairl** [comment]
same here... may be a incident ?
status means everything ok...

**Comment by ossanna16** [comment, marked as answer]
Hi there @casmatos and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
